platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ApplicationDefinitionLoader.java

```diff
@@ -51,21 +50,21 @@ public final class ApplicationDefinitionLoader {
 
     private ApplicationDefinitionLoader() {}
 
+    public static ApplicationDefinition loadDefault(final Path configPath) throws ConfigurationException {
+        return load(LegacyConfigPropertiesLoader.loadConfigFile(configPath));
+    }
+
     /**
      * Parses the configuration file specified by the {@link PathsConfig#getConfigPath()} setting, configures all
      * appropriate system settings, and returns a generic {@link ApplicationDefinition}.
      *
-     * @param localNodesToStart
-     * 		the {@link Set} of local nodes to be started, if specified
      * @return an {@link ApplicationDefinition} specifying the application to be loaded and all related configuration
      * @throws ConfigurationException
      * 		if the configuration file specified by {@link PathsConfig#getConfigPath()} does not exist
      */
-    public static ApplicationDefinition load(
-            @NonNull final LegacyConfigProperties configurationProperties, @NonNull final Set<NodeId> localNodesToStart)
+    public static ApplicationDefinition load(@NonNull final LegacyConfigProperties configurationProperties)
```

Comment:
@NonNull for method return value missing

Reply:
added